drivers/bluetooth/hci/Kconfig.silabs

```diff
@@ -33,19 +33,193 @@ config BT_SILABS_EFR32_COMPLETED_PACKETS_THRESHOLD
+config BT_SILABS_EFR32_PAWR_PACKET_REQUEST_ADVANCE
+	int "Number of subevents in advance to request PAwR data"
+	default 1
+	range 1 6
+	depends on BT_PER_ADV_RSP
+	help
+	  The controller asks the host to provide data for upcoming subevents
+	  before the subevents are sent over the air. This parameter is used to
+	  configure how many subevents in advance the requests are sent to the
+	  host. The bigger the advance is, the more time the host has to
+	  process the request, but the controller has to reserve more memory
+	  for buffering the subevent data. Note that the maximum advance is
+	  limited by the number of subevents configured for a PAwR train.
+	  Therefore, the actual advance may be less than the number configured
+	  in this parameter.
+
+config BT_SILABS_EFR32_ACTIVATE_POWER_CONTROL
+	bool "EFR32 LE Power Control"
+	default y
+	depends on BT_TRANSMIT_POWER_CONTROL
+	help
+	  Enable power control to adjust the transmit power based on the
+	  received signal strength. The controller will ask the transmitter
+	  to adjust its transmit power to keep the RSSI of the received
+	  packets within the golden ranges, specified separately for each PHY.
+
+config BT_SILABS_EFR32_GOLDEN_RSSI_MIN_1M
+	int "Lower bound of the golden RSSI range for 1M PHY"
+	default -60
+	depends on BT_TRANSMIT_POWER_CONTROL
+	help
+	  The lower bound of the golden RSSI range for 1M PHY.
+
+config BT_SILABS_EFR32_GOLDEN_RSSI_MAX_1M
+	int "Upper bound of the golden RSSI range for 1M PHY"
+	default -40
+	depends on BT_TRANSMIT_POWER_CONTROL
+	help
+	  The upper bound of the golden RSSI range for 1M PHY.
+
+config BT_SILABS_EFR32_GOLDEN_RSSI_MIN_2M
+	int "Lower bound of the golden RSSI range for 2M PHY"
+	default -60
+	depends on BT_TRANSMIT_POWER_CONTROL
+	help
+	  The lower bound of the golden RSSI range for 2M PHY.
+
+config BT_SILABS_EFR32_GOLDEN_RSSI_MAX_2M
+	int "Upper bound of the golden RSSI range for 2M PHY"
+	default -40
+	depends on BT_TRANSMIT_POWER_CONTROL
+	help
+	  The upper bound of the golden RSSI range for 2M PHY.
+
+config BT_SILABS_EFR32_GOLDEN_RSSI_MIN_CODED_S8
+	int "Lower bound of the golden RSSI range for Coded S=8 PHY"
+	default -60
+	depends on BT_TRANSMIT_POWER_CONTROL
+	help
+	  The lower bound of the golden RSSI range for Coded S=8 PHY.
+
+config BT_SILABS_EFR32_GOLDEN_RSSI_MAX_CODED_S8
+	int "Upper bound of the golden RSSI range for Coded S=8 PHY"
+	default -40
+	depends on BT_TRANSMIT_POWER_CONTROL
+	help
+	  The upper bound of the golden RSSI range for Coded S=8 PHY.
+
+config BT_SILABS_EFR32_GOLDEN_RSSI_MIN_CODED_S2
+	int "Lower bound of the golden RSSI range for Coded S=2 PHY"
+	default -60
+	depends on BT_TRANSMIT_POWER_CONTROL
+	help
+	  The lower bound of the golden RSSI range for Coded S=2 PHY.
+
+config BT_SILABS_EFR32_GOLDEN_RSSI_MAX_CODED_S2
+	int "Upper bound of the golden RSSI range for Coded S=2 PHY"
+	default -40
+	depends on BT_TRANSMIT_POWER_CONTROL
+	help
+	  The upper bound of the golden RSSI range for Coded S=2 PHY.
```

Comment:
These all have the same single dependency, so they should likely be inside a single `if .. endif` pair.